src/main/resources/static/app/request/RequestController.js

```diff
@@ -641,13 +641,18 @@ angular.module('ProcessApp.controllers')
                 return _.includes($scope.excerptSelectionStates, state);
             };
 
+            $scope.isAssignedAsPathologist = function (req) {
+                return req? req.pathologistEmail === $scope.globals.currentUser.username : false;
+            };
+
             /**
              * Return true iff the curent user is a requester and the status is 'Open'
              * of the current user is a Palga user and the status is one of the
              * statuses where editing is allowed ('Open', 'Review', 'Approval').
              */
-            $scope.isEditStatus = function(status) {
-                return ($scope.isRequester() && status === 'Open') ||
+            $scope.isEditStatus = function (status) {
+                // check if current user is assigned as pathologist for selected request
+                return ($scope.isRequester() && status === 'Open' && !$scope.isAssignedAsPathologist($scope.request)) ||
```

Comment:
What if the pathologist is the requester that created the request? It may be better to just check that {{$scope.request.requesterId === $scope.globals.currentUser.username}} or something like that.